.bazelrc

```diff
@@ -0,0 +1,2 @@
+# Required by abseil-cpp.
+common --cxxopt=-std=c++14
```

Comment:
In Protobuf we used `--per_file_copt=.*/absl/strings/string_view.h@-Wno-unreachable-code`

Reply:
Added

Reply:
I meant instead of --cxxopt.

Reply:
Hmm, then I'm not sure what that would do. The failure I see without this is a preprocessor assert on the std version.